pkg/controller/service/service_controller.go

```diff
@@ -598,6 +598,10 @@ func getNodeConditionPredicate() corelisters.NodeConditionPredicate {
 			return false
 		}
 
+		if _, hasExcludeBalancerLabel := node.Labels[constants.LabelNodeRoleExcludeBalancer]; hasExcludeBalancerLabel {
```

Comment:
I just opened https://github.com/kubernetes/kubernetes/issues/53341. We shouldn't depend on cmd/kubeadm from core controllers.